test/apple_shell_testutils.sh

```diff
@@ -423,8 +423,8 @@ function do_action() {
        "$@" \
   )
 
-  echo "Executing: bazel ${action} ${bazel_options[*]}" > "$TEST_log"
-  bazel "${action}" "${bazel_options[@]}" > "$TEST_log" 2>&1
+  echo "Executing: bazel ${action} ${bazel_options[*]}" >> "$TEST_log"
+  bazel "${action}" "${bazel_options[@]}" >> "$TEST_log" 2>&1
```

Comment:
It was here that I noticed this. I'm not sure if all of these should be `>>`, or if one should be `>` to start anew. I defaulted to making all redirects to `$TEST_log` be appending.

Reply:
good find! it should be `>>` for all of them